modules/cas_cache/layer_cache_management.c

```diff
@@ -2555,6 +2554,8 @@ int cache_mngt_init_instance(struct ocf_mngt_cache_config *cfg,
 
 	return result;
 err:
+	cmd->min_free_ram = context->min_free_ram;
```

Comment:
This value won't be set before line 2511, so we might be leaking 8B of uninitialized kernel memory here.

Reply:
`context` is allocated with `kzalloc()` so it is initialized

Reply:
right, I must've overlooked that